.github/workflows/snap.yml

```diff
@@ -0,0 +1,21 @@
+name: Build test Amber Snap
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
```

Comment:
Hang on. Why are we building it on each master merge and not when we create a new version release?

Reply:
This will ensure each commit isn't breaking the snap package. We're nit releasing the snap though. We don't need to build it with each release that can be done in the website itself

Reply:
Okay. This makes sense then.